sycl/test/regression/mad_sat.cpp

```diff
@@ -0,0 +1,138 @@
+// RUN: %clangxx -fsycl %s -o %t.out
```

Comment:
This change is already comitted with intel/llvm#2790 and is irrelevant to the patch.
Could you, please, remove it from the patch?